tests/dummy/app/templates/demos/sl-menu.hbs

```diff
@@ -9,205 +9,115 @@
     <div class="col-lg-6">
         <h6>Template</h6>
         <pre>\{{sl-menu
-    actionInitiated="actionInitiatedHandler"
-    changeRoute="changeRouteHandler"
-    keyEvents=controller.keyHandler
-    keyboardInUse=controller.keyboardInUse
-    menu=controller.content
-    selectionMade="selectionMadeHandler"
-    showAll="true"
+    action="doSomething"
+    allowShowAll=true
+    items=menuItems
+    streamName=menuStreamName
 }}</pre>
     </div>
 
     <div class="col-lg-6">
         <h6>Rendered Component</h6>
         {{sl-menu
-            actionInitiated="actionInitiatedHandler"
-            changeRoute="changeRouteHandler"
-            keyEvents=controller.keyHandler
-            keyboardInUse=controller.keyboardInUse
-            menu=controller.content
-            selectionMade="selectionMadeHandler"
-            showAll="true"
+            action="doSomething"
+            allowShowAll=true
+            items=menuItems
+            streamName=menuStreamName
         }}
     </div>
 </div>
 
-<br>
-<div class="row">
-    <div class="col-lg-12">
-        <p>In addition to being able to interact with the menu via a mouse, the following keyboard shortcuts can also be used:</p>
```

Comment:
I get that these specific instructions might not fit with the update documentation on this page, but shouldn't we give the user a tip about how they can demo the stream setup via the use of certain keys related to this demo?  Otherwise how will they know that they should dig into the dummy app controllers to see we have it setup and then they can try it?